scripts/eval/alpaca_farm.sh

```diff
@@ -1,4 +1,26 @@
-# example script for alpacafarm
-export OPENAI_API_KEY=YOUR_API_KEY
+# python -m eval.alpaca_farm.run_eval \
```

Comment:
Don't you still need the openai key?

Reply:
That is expected to be set in the environment. I want to delete this because if it's written in the code, we have to comment it out before committing and pushing it to github. Let me add a comment.